doc/tutorials/content/sources/don_segmentation/don_segmentation.cpp

```diff
@@ -177,12 +177,12 @@ main (int argc, char *argv[])
       cloud_cluster_don->points.push_back (doncloud->points[*pit]);
     }
 
-    cloud_cluster_don->width = int (cloud_cluster_don->points.size ());
+    cloud_cluster_don->width = int (cloud_cluster_don->size ());
```

Comment:
Can we remove this cast? I don't think it will trigger warnings on anything but MSVC.

Reply:
Should the casts be another PR or this one?

Reply:
Roughly 2000 matches of `size())` with `static_cast` or `int(`

Reply:
Just address in this PR the lines you're already changing. Everything else can go on a separate PR.